### PR TITLE
case-lib/lib.sh: use sudo to access debugfs files

### PR DESCRIPTION
**Pull request by plbossart**
Apparently debugfs requires more care. Oh well.

Link: https://github.com/thesofproject/sof-test/issues/1171

### CODE REVIEW

File: case-lib/lib.sh [hunk 1]
```diff
@@ -831,7 +831,7 @@ is_ipc4()
     fi
 
     dlogw "using pre kernel 6.9 backwards-compatible fallback with ipc_type kernel parameter"
-    if ipc_type=$(cat /sys/module/snd_sof_pci/parameters/ipc_type); then
+    if ipc_type=$(sudo cat /sys/module/snd_sof_pci/parameters/ipc_type); then
```
Comment (marc-hb) [inline case-lib/lib.sh:834]: This does not seem required. When running sof_insert/sof_remove I noticed `sudo` can be pretty slow, even slower on Fedora I think. Of course we should ideally cache the result of `is_ipc4()` but right now we don't.

**Review by marc-hb — APPROVED** [review]
Tests results look good. Very small change that gets rid of very confusing error messages: merging.

**Comment by marc-hb** [comment]
@plbossart want to keep this a draft still? If not then please merge

**Comment by plbossart** [comment]
I missed one:
````
grep: /sys/kernel/debug/sof/fw_profile/*: Permission denied
````

**Review by marc-hb — APPROVED** [review]

**Review by kv2019i — APPROVED** [review]

**Pull request merged**